### DISCUSSION TITLE
A few considerations about the new notebook

### DISCUSSION THREAD

**Discussion by esanzgar** [hypothesis/client#3013] [Ideas]
A few observation about the notebook project:
- how do I switch between groups?
- how do I filter by URL or document?
- would reusing elements from the client and the activity pages make the interface simpler to use?

Reusing elements from the activity pages:

![image](https://user-images.githubusercontent.com/8555781/107957133-71d27300-6fa0-11eb-9b01-780bff23c020.png)

![image](https://user-images.githubusercontent.com/8555781/107957236-9e868a80-6fa0-11eb-9116-e73556eba562.png)

Reusing elements from the client:

![image](https://user-images.githubusercontent.com/8555781/107957351-c544c100-6fa0-11eb-9a5e-9d97ef2e3644.png)

**Comment by lyzadanger** [comment]
Hi @esanzgar

To avoid re-treading some water that has already been examined, please see the existing Balsamic sketches for the Notebook (requires H login). Particularly some of the sections related to facets (like the document and group facets). These sketches do not yet fully accommodate narrow viewports (mea culpa for not mobile-first-ing this). For example, there is a sketch:

![image](https://user-images.githubusercontent.com/439947/107958186-a7129d00-6f6f-11eb-9c86-95f8fc93ab62.png)

Showing some elements of a multi-faceted interface.

None of these sketches have high fidelity, and they are loose, but some thinking has started happening here...
